### DISCUSSION TITLE
Erro au tentar rodar o comando - dotnet ef migrations add InitialCreation

### DISCUSSION THREAD

**Discussion by etaptecnologia** [balta-io/forum#180] [Tira Dúvidas] [answered]
Boa tarde pessoal.

Eu estou usando a IDE do Visual Studio e não sei se isso interfere, mas estou tendo o seguinte erro.

Não foi possível executar porque o comando ou o arquivo especificado não foi encontrado.
Possíveis motivos para isso incluem:
  * Você digitou incorretamente um comando de dotnet interno.
  * Você pretendia executar um programa .NET, mas dotnet-ef não existe.
  * Você pretendia executar uma ferramenta global, mas não foi possível encontrar um executável com prefixo de dotnet com esse nome no CAMINHO.

**Comment by andrebaltieri** [comment, marked as answer]
@etaptecnologia por favor, siga como está no curso... futuramente você vai precisar desses comandos na parte de DevOps também.
